# Sourabh Kumar

**Bioinformatics Scientist | AIIMS Delhi**  
üìß bioinfosourabh@gmail.com  
üåê [LinkedIn](https://linkedin.com/in/sourabh-kumar-bioinfo)  ‚Ä¢ [ORCID](https://orcid.org/)  
üì∏ Outside of research, I also pursue photography/filmaking .

---

## My tech tools üíª

![Python](https://img.shields.io/badge/python-3670A0?style=flat-square&logo=python&logoColor=ffdd54)
![R](https://img.shields.io/badge/r-%23276DC3.svg?style=flat-square&logo=r&logoColor=white)
![Shell Script](https://img.shields.io/badge/shell_script-%23121011.svg?style=flat-square&logo=gnu-bash&logoColor=white)
![GNU-Bash](https://img.shields.io/badge/GNU%20Bash-4EAA25?style=flat-square&logo=GNU%20Bash&logoColor=white)
![MySQL](https://img.shields.io/badge/mysql-%2300f.svg?style=flat-square&logo=mysql&logoColor=white)
![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=flat-square&logo=javascript&logoColor=%23F7DF1E)

![Pandas](https://img.shields.io/badge/pandas-%23150458.svg?style=flat-square&logo=pandas&logoColor=white)
![NumPy](https://img.shields.io/badge/numpy-%23013243.svg?style=flat-square&logo=numpy&logoColor=white)
![scikit-learn](https://img.shields.io/badge/scikit--learn-%23F7931E.svg?style=flat-square&logo=scikit-learn&logoColor=white)
![PyTorch](https://img.shields.io/badge/PyTorch-%23EE4C2C.svg?style=flat-square&logo=PyTorch&logoColor=white)
![Keras](https://img.shields.io/badge/Keras-%23D00000.svg?style=flat-square&logo=Keras&logoColor=white)
![Weight-and-Biases](https://img.shields.io/badge/Weights_&_Biases-FFBE00?style=flat-square&logo=WeightsAndBiases&logoColor=white)
![Plotly](https://img.shields.io/badge/Plotly-%233F4F75.svg?style=flat-square&logo=plotly&logoColor=white)
![SciPy](https://img.shields.io/badge/SciPy-%230C55A5.svg?style=flat-square&logo=scipy&logoColor=%white)
![Docker](https://img.shields.io/badge/docker-%230db7ed.svg?style=flat-square&logo=docker&logoColor=white)
![GitHub Actions](https://img.shields.io/badge/github%20actions-%232671E5.svg?style=flat-square&logo=githubactions&logoColor=white)

![GitHub](https://img.shields.io/badge/github-%23121011.svg?style=flat-square&logo=github&logoColor=white)
![Linux](https://img.shields.io/badge/Linux-FCC624?style=flat-square&logo=linux&logoColor=black)
![Visual Studio Code](https://img.shields.io/badge/Visual%20Studio%20Code-0078d7.svg?style=flat-square&logo=visual-studio-code&logoColor=white)
![Neovim](https://img.shields.io/badge/NeoVim-%2357A143.svg?&style=flat-square&logo=neovim&logoColor=white)
![AWS](https://img.shields.io/badge/AWS-%23FF9900.svg?style=flat-square&logo=amazon-aws&logoColor=white)
![Anaconda](https://img.shields.io/badge/Anaconda-%2344A833.svg?style=flat-square&logo=anaconda&logoColor=white)
![google-colab](https://img.shields.io/badge/Colab-F9AB00?style=flat-square&logo=googlecolab&color=525252)

--------------------------------------------------------------------------------


## Research Focus

I specialize in high-throughput genomic analysis, precision diagnostics, and cloud-native bioinformatics. My work spans the development of scalable pipelines for whole exome/genome sequencing, machine learning models for variant prioritization, and integrative multi-omics frameworks for pediatric tumors and rare genetic disorders.

---

## Areas of Expertise

- **Computational Genomics**: WES, WGS, CNV/SV detection, somatic/germline variant calling  
- **NGS Pipeline Development**: Nextflow, Docker/Singularity, AWS, HPC environments  
- **Machine Learning in Genomics**: XGBoost, GNNs, Deep Learning, AI-based variant scoring  
- **Structural Bioinformatics**: Molecular dynamics (GROMACS), mutational modeling, docking  
- **Multi-Omics Integration**: scRNA-Seq, RNA-Seq, 16S metagenomics, ChIP-Seq  

---

## Projects

- [`TrioExome-Analysis`](https://github.com/bioinfosourabh/TrioExome-Analysis):  
  Clinical pipeline for trio-based whole exome sequencing with automated annotation and priortization.

- [`Variant-Prioritizer`](https://github.com/bioinfosourabh/Variant-Prioritizer):  
  Framework for deleterious variant classification integrating CADD, SIFT, and dbNSFP.

- [`WGS-AMR`](https://github.com/bioinfosourabh/WGS-AMR-Ecoli):  
  Whole genome sequencing pipeline for identifying antimicrobial resistance markers.

- [`Single-Cell-Analysis`](https://github.com/bioinfosourabh/Single-Cell-Analysis):  
  Single Cell Data Analysis Workflow using the Seurat package



---

## Publications

1.	Kumar, Sourabh, et al. "Exploring Familial Hypospadias: Genetic Insights from Copy Number Variants in a Quad Family." (2024). [https://doi.org/10.21203/rs.3.rs-4843906/v1]

2.	Phugat, S., Sharma, J., Kumar, S., Jain, V., Dhua, A. K., Yadav, D. K., ... & Goel, P. (2024). Genetic landscape of congenital pouch colon: systematic review and functional enrichment study. Pediatric Surgery International, 40(1), 314.

3.	Goel, P., Sharma, M., Kaushik, H., Kumar, S., Singh, H., Jain, V., ... & Agarwala, S. (2024). Genetic Markers of Spina Bifida in an Indian Cohort. Journal of Indian Association of Pediatric Surgeons, 29(5), 529-535.

4.	Sharma, J., Sharma, M., Kumar, S., Kaushik, H., Pandey, H., Lal, D., ... & Goel, P. (2025). Genetic Markers of Spina Bifida: Enrichment of Pathogenic Variants and Variants of Uncertain Significance. Journal of Indian Association of Pediatric Surgeons, 30(2), 163-169.

5.	Kumar, S., Sharma, J., Sardar, R., Jain, V., Dhua, A. K., Yadav, D. K., ... & Goel, P. (2025). KMT2C Polymorphism in Familial Hypospadias. Indian Journal of Pediatrics, 1-3

6.	Goel, Prabudh, et al. "Chromosomal Microarray Analysis in Spina Bifida: Genetic Heterogeneity and Its Clinical Implications".  Journal of Indian Association of Pediatric Surgeons
---

## Contact

Feel free to connect .  
üì¨ bioinfosourabh@gmail.com  
